Lib/test/test_inspect/test_inspect.py

```diff
@@ -811,6 +811,15 @@ def test_getfile_builtin_module(self):
             inspect.getfile(sys)
         self.assertStartsWith(str(e.exception), '<module')
 
+    def test_getfile_custom_module(self):
+        import re
+        custom_module = types.ModuleType('custom_module')
+        msg = re.escape(
+             f'Cannot get source from {custom_module!r}'
+        )
+        with self.assertRaisesRegex(TypeError, msg) as e:
+            inspect.getfile(custom_module)
```

Comment:
```suggestion
        my_mod = types.ModuleType('my_mod')
        msg = re.escape(f'Cannot get source from {my_mod!r}')
        with self.assertRaisesRegex(TypeError, msg) as e:
            inspect.getfile(my_mod)
```

And move `import re` at the top of the module if it's not already imported.

Reply:
Don't forget the `import re`.